shim.go

```diff
@@ -30,7 +30,32 @@ type ShimInfo struct {
 	Port  string
 }
 
-func startShim(pod *vc.Pod) (int, error) {
+func startShim(info interface{}) (int, error) {
+	switch obj := info.(type) {
+	case *vc.Pod:
+		return startShimFromPod(obj)
+	case *vc.Process:
+		return startShimFromProcess(obj)
+	default:
+		return -1, fmt.Errorf("Invalid interface, could not start the shim")
+	}
+}
+
+func startShimFromProcess(process *vc.Process) (int, error) {
+	cmd := exec.Cmd{
+		Path: defaultShimPath,
+		Args: []string{"-t", process.Token, "-s", defaultProxyIP, "-p", defaultProxyPort},
+		Env:  os.Environ(),
+	}
+
+	if err := cmd.Start(); err != nil {
+		return -1, err
+	}
+
+	return cmd.Process.Pid, nil
+}
+
```

Comment:
Yes I am okay with that. But don't you think we should do that in a following PR as a cleaning purpose ? Because I will have to modify create.go while this PR si supposed to be the exec() implementation ?